static/app/views/alerts/rules/metric/details/metricChart.tsx

```diff
@@ -532,47 +528,13 @@ class MetricChart extends PureComponent<Props, State> {
       location,
       isOnDemandAlert,
     } = this.props;
-    const {aggregate, timeWindow, environment, dataset} = rule;
-
-    // Fix for 7 days * 1m interval being over the max number of results from events api
-    // 10k events is the current max
-    if (
-      timePeriod.usingPeriod &&
-      timePeriod.period === TimePeriod.SEVEN_DAYS &&
-      interval === '1m'
-    ) {
-      timePeriod.start = getUtcDateString(
-        // -5 minutes provides a small cushion for rounding up minutes. This might be able to be smaller
-        moment(moment.utc(timePeriod.end).subtract(10000 - 5, 'minutes'))
-      );
```

Comment:
The only change here is that previously we were modifying the prop directly which React isn't a fan of. I've changed it to copy the original prop into a new object instead.